test/hotel_organizer_test.rb

```diff
@@ -0,0 +1,102 @@
+require_relative 'test_helper'
+
+describe "HotelOrganizer Class" do  
+  before do
+    @hotel_organizer = Hotel::HotelOrganizer.new
+    
+    @reservation1 = @hotel_organizer.make_reservation(18, Date.new(2018,01,01), Date.new(2018,01,06), nil)
```

Comment:
Consider storing your dates as variable so it is clear how the dates relate to each other and what you are testing.
```suggestion
    start_date1 = Date.new(2018,01,01)
    end_date1 = start_date1 + 5
    
    start_date2 = start_date1 + 2
    end_date2 = start_date2 + 9
    
    ...
```

Reply:
Added refactors.txt file and roadmap to future changes.